### DISCUSSION TITLE
対人オンラインを作ることは可能でしょうか？

### DISCUSSION THREAD

**Discussion by john-banjirou** [Equim-chan/Mortal#37] [Ideas]
こんにちは

オンライン対人麻雀、を作りたいと考えています。

このmortalを使って、作ることは可能でしょうか？

・python　部分だけを改変
・rust、部分はいじらない

これが、理想の条件です。

よろしくおねがいします。

万二郎

**Comment by Equim-chan** [comment]
No, it is not possible. libriichi, the core library that implements logic of the mahjong game, is entirely written in Rust.

It is also not ideal. Mortal's goal is simply to be an AI, and all of the Python and Rust code serves no other purpose than that. Some Rust code in libriichi may be useful and could be reused, but it is not designed for creating an online mahjong platform for humans in the first place.

**Reply by john-banjirou** [reply]
right. I do understand. it is just big work to create mahjong game and take advantage anything.
thanks.
